lambda/rest-endpoints/src/app.ts

```diff
@@ -0,0 +1,45 @@
+import bodyParser from 'body-parser';
+import {renderFile as ejs} from "ejs";
+import express from 'express';
+import {getBodyContentAsJson} from "./helpers";
+import {importNewData} from "./submit-data";
+
+export const app = express();
+app.set('view engine', 'ejs');
+app.engine('.ejs', ejs);
+
+const router = express.Router();
+router.use(bodyParser.json());
+
+router.post('/api/curation', (req, resp)=>{
+  const textContent = getBodyContentAsJson(req.body);
+
+  if(req.header("Content-Type") != "application/json") {
```

Comment:
I'll stop mentioning but the non-triple-equalses are making me itchy 😀  – non-blocking, we can sort in one pass with a `format:write` down the line, but enforcing will eliminate a class of bug